packages/semcom-core/lib/component/models/component-data-types.ts

```diff
@@ -0,0 +1,9 @@
+import { Quad } from 'rdf-js';
+
+export interface ComponentDataTypes {
+  quads: Quad[];
+  string: string;
```

Comment:
Maybe use 'text' instead of 'string' in alignment with default fetch response functions? 
```suggestion
  text: string;
```